.circleci/config.yml

```diff
@@ -303,19 +307,18 @@ commands:
 
   compile:
     steps:
+      - run:
+          name: Pre building shared libraries
+          command: NODE_OPTIONS="--max-old-space-size=7168" npx nx run-many -t build --projects=tag:scope:shared:lib --parallel=2
+          environment:
+            NODE_ENV: test
+            CI: false
```

Comment:
(curious) Why CI: false?

Reply:
Ah, good call out. I need to file a follow up for this. Typescript will actually fail on warnings when CI is set to true, and I couldn't address all the warnings in this PR.

Reply:
Follow up added: [FXA-8215](https://mozilla-hub.atlassian.net/browse/FXA-8215).